langchain/src/callbacks/handlers/tracers.ts

```diff
@@ -55,8 +75,33 @@ export interface ToolRun extends BaseRun {
 
 export type Run = LLMRun | ChainRun | ToolRun;
 
+export interface RunV2Base {
+  id: string; // v4 uuid
```

Comment:
Why would we care what kind of uuid it is here

Reply:
Not strongly. Meant to differentiate between the number `id` of the v1 run. I can remove the comments entirely if that's cleaner